src/PinoLogger.ts

```diff
@@ -140,6 +141,17 @@ export class PinoLogger implements PinoMethods {
     return storage.getStore()?.logger || outOfContext!;
   }
 
+  public getRequestContext(): Request {
+    const store = storage.getStore();
+    const requestContext = storage.getStore()?.logger?.bindings()?.req;
```

Comment:
does the existing [logger getter](https://github.com/iamolegga/nestjs-pino/blob/master/src/PinoLogger.ts#L137-L141) fit your needs?